### ISSUE TITLE
Incorrect view engine extension

### ISSUE THREAD

**Issue by mlbonniec** [pmb0/express-tsx-views#70]
Hi, I absolutely don't understand [this line](https://github.com/pmb0/express-tsx-views/blob/9e1dd7ff4d3f19ae0db78b50c583788c1db704f9/src/react-view-engine.ts#L25), because once compiled, the library is in `.js`, so `js` will be assigned to this variable, and the view engine will be incorrect.  
Can you explain it to me?
Because I get an error, it can't find my views. Which I solved by simply replacing the line with this `const extension = 'tsx';`

**Comment by pmb0** [comment]
If compiled and started with `node`, this variable is `js`. If using things like ts-node, it is `tsx`. How do you run your app?

**Comment by mlbonniec** [comment]
I'm starting my app with `nodemon` and `ts-node`, but the variable is set to `js`
```bash
nodemon -r dotenv/config --exec ts-node --files ./src/app.ts
```

**Comment by pmb0** [comment]
Hi, i can't reproduce. I tried the example app in this repo:

```bash
$ git clone git@github.com:pmb0/express-tsx-views.git
$ cd express-tsx-views
$ yarn
$ nodemon -r dotenv/config --exec ts-node --files example/bin/www.ts
```

Every worked fine. Can you provide a repo that shows the problem?

**Comment by pmb0** [comment]
@mlbonniec In the meantime I could understand the problem. It will be fixed shortly by a new version of `express-tsx-views`.

**Comment by pmb0** [comment]
@mlbonniec  Fixed in [v1.2.5](https://github.com/pmb0/express-tsx-views/blob/master/CHANGELOG.md#125-2020-12-13)
